src/assets/scripts/client/game/GameController.js

```diff
@@ -77,12 +76,50 @@ export default class GameController {
      * @for GameController
      * @method init_pre
      */
-    init_pre() {
+    init_pre(getDeltaTime) {
+        this.getDeltaTime = getDeltaTime;
+
+        // TODO: move calling of these methods to the proper lifecycle positions
         this.setupHandlers();
+        this.enable();
         this.initializeEventCount();
     }
 
     /**
+    * Initialize blur functions used during game pausing
+    * @for GameController
+    * @method setupHandlers
+    * @return
+    */
+    setupHandlers() {
+        // Set blurring function
+        $(window).blur(() => {
+            this.game.focused = false;
+        });
```

Comment:
You're okay with using jQuery here? I thought I remembered there was a reason we wanted to eliminate it (though that may not be possible?)